src/CattyTests/Firebase/FirebaseCrashlyticsReporterTests.swift

```diff
@@ -21,54 +21,60 @@
  */
 
 import Firebase
+
 @testable import Pocket_Code
+
 import XCTest
 
-final class FirebaseCrashlyticsSetupTests: XCTestCase {
+final class FirebaseCrashlyticsReporterTests: XCTestCase {
 
-    var app: AppDelegate?
     var crashlytics: CrashlyticsMock?
+    var reporter: FirebaseCrashlyticsReporter?
 
     override func setUp() {
         super.setUp()
 
         crashlytics = CrashlyticsMock(collectionEnabled: false)
-        app = AppDelegateMock(crashlytics: crashlytics!)
+        reporter = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
+    }
+
+    func testAppDelegateInitialization() {
```

Comment:
In my opinion this test is not really interesting. It would be more interesting to test, that after the `application..didFinishLaunchingWithOptions` is called, that the two properties are actually set. But since this is not available in DEBUG mode, I think this is very difficult.

Reply:
so should I just remove the test?